docs/env.md

```diff
@@ -1,117 +1,260 @@
+  - `"test-marsha-source"` in test;
+  - `"staging-marsha-source"` in staging;
+  - `"preprod-marsha-source"` in preproduction;
+  - `"production-marsha-source"` in production.
+
+#### DJANGO_CLOUDFRONT_ACCESS_KEY_ID
+
+The ID of the AWS master account with which we want to sign urls (it is declared in the CloudFront distribution as a signing account).
+
+Note: Must be associated with a master account as IAM accounts cannot sign URLs.
+
+- Type: string
+- Required:
+  - Yes when `DJANGO_CLOUDFRONT_SIGNED_URLS_ACTIVE` is `True`;
+  - No otherwise.
+- Default: None;
+
+#### CLOUDFRONT_PRIVATE_KEY_PATH
+
+Path to a private key corresponding to the id in `DJANGO_CLOUDFRONT_ACCESS_KEY_ID`. Also used to sign Cloudfront URLs.
+
+- Type: string
+- Required:
+  - Yes when `DJANGO_CLOUDFRONT_SIGNED_URLS_ACTIVE` is `True` and the key is not located in the default path;
+  - No otherwise.
+- Default: `{REPO_ROOT}/.ssh/cloudfront_private_key`
+
+#### DJANGO_CLOUDFRONT_SIGNED_URLS_ACTIVE
+
+Whether Cloudfront URLs for MP4 files and subtitle tracks should be cryptographically signed.
+
+Note: Preview images are never signed as a matter of policy; adaptive streaming formats pose technical challenges when it comes to signed URLs, so we're not doing any signing there for now.
+
+- Type: Boolean
+- Required: No
+- Default: Varies depending on the environment:
+  - `False` in development and test;
+  - `True` in all other environments.
+- Choices: `True` or `False`
+
+#### DJANGO_CLOUDFRONT_URL
+
+The URL for the AWS Cloudfront distribution for the relevant AWS deployment. This is the domain that will be used to distribute processed files to end users.
+
+- Type: string
+- Required: Yes
+- Default: None
+
+
+## 2. AWS deployment environment
+
+Specified in `{REPO_ROOT}/src/aws/env.d/development`.
+
+#### AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY
+
+A key ID + secret pair for an AWS IAM account with administrative access to the resources the AWS deployment we're operating on is using.
+
+- Type: string
+- Required: Yes
+- Default: None
+
+#### TF_VAR_aws_region
+
+The Amazon Web Services region where we deployed or want to deploy our serverless stack.
+
+- Type: string
+- Required: No
+- Default: `"eu-west-1"`
+- Choices: Any valid AWS region name.
+
+#### TF_VAR_cloudfront_access_key_id
+
+The ID of the AWS master account with which we want to sign urls (it is declared in the CloudFront distribution as a signing account).
+
+- Type: string
+- Required: Yes
+- Default: None
+
+#### TF_VAR_update_state_secret
+
+Secret used to sign messages exchanged between the Django backend & AWS lambdas.
+
+Note: should be included in the list the value from `DJANGO_UPDATE_STATE_SHARED_SECRETS` for the Django backend deployment with which our deployment's lambdas will communicate.
+
+- Type: string
+- Required: Yes
+- Default: None
+
+#### TF_VAR_update_state_endpoint
+
+Location for the Django backend endpoint for state updates our lambdas should POST to when they process files.
```

Comment:
URL of the endpoint in Marsha to which our lambdas should POST state updates when they process files.